_config.yml

```
@@ -50,3 +52,26 @@ defaults:
           link: https://www.canada.ca/en/government/about/design-system.html
         - title: Information architecture specification
           link: https://design.canada.ca/architecture/canada-content-information-architecture-specification.html
+  - scope:
+      path: "common-design-patterns/*.md"
+    values:
+      breadcrumbs:
+        - title: About Canada.ca
+          link: https://www.canada.ca/en/government/about.html
+        - title: Canada.ca design system
+          link: https://www.canada.ca/en/government/about/design-system.html
+        - title: Template and pattern library for Canada.ca
+          link: https://www.canada.ca/en/government/about/design-system/pattern-library.html
+  - scope:
+      path: "common-design-patterns/global-footer/*.md"
+    values:
+      breadcrumbs:
+        - title: About Canada.ca
+          link: https://www.canada.ca/en/government/about.html
+        - title: Canada.ca design system
+          link: https://www.canada.ca/en/government/about/design-system.html
+        - title: Template and pattern library for Canada.ca
+          link: https://www.canada.ca/en/government/about/design-system/pattern-library.html
+        - title: Global footer
+          link: https://design.canada.ca/common-design-patterns/site-footer.html
+      permalink: /common-design-patterns/:basename:output_ext
```

```suggestion
  - scope:
      path: "common-design-patterns/global-footer/*.md"
    values:
      breadcrumbs:
        - title: About Canada.ca
          link: https://www.canada.ca/en/government/about.html
        - title: Canada.ca design system
          link: https://www.canada.ca/en/government/about/design-system.html
        - title: Template and pattern library for Canada.ca
          link: https://www.canada.ca/en/government/about/design-system/pattern-library.html
        - title: Global footer
          link: https://design.canada.ca/common-design-patterns/site-footer.html
      permalink: /common-design-patterns/:basename:output_ext
```
Only this change should be apply to the config.yml. The rest is already in the master branch file. It will avoid conflict when comes to merge.

---

Okay